pages/pipelines/command_step.md.erb

```diff
@@ -241,8 +241,36 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
+If you retry a job, the information about the failed job(s) remains, and a new job is created. The history of retried jobs is preserved and immutable.
 
-## Automatic retry attributes
```

Comment:
Because we're downgrading these headings, how about linking each element in this table 
![image](https://user-images.githubusercontent.com/95874/167170272-b79d8c9c-6584-40a0-bff3-a7239432325b.png)

to the relevant subheading?